### PR TITLE
add etherpad stage infra (mysql db, secrets)

### PR DESCRIPTION
**Pull request by cmharlow**
Jira: https://mozilla-hub.atlassian.net/browse/SE-2125

What this PR does:
* adds Etherpad infra for Etherpad stage, based on / migrated from https://github.com/mozilla-it/etherpad-infra

### CODE REVIEW

File: terraform/applications/etherpad/state.tf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  }
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws",
+      version = "~> 3.0"
+    }
+  }
+
+  required_version = ">= 0.12"
```
Comment (smarnach) [inline terraform/applications/etherpad/state.tf:15]: This should be 1.0.4, to match the tfenv version.

File: terraform/applications/etherpad/variables.tfvars [hunk 2]
```diff
@@ -0,0 +1,6 @@
+cost_center   = "1440"
```
Comment (smarnach) [inline terraform/applications/etherpad/variables.tfvars:1]: I suggest deleting either this file or the defaults in variables.tf – having the same values in both places seems redundant.

**Review by smarnach — APPROVED** [review]
Looks good to me.

File: terraform/applications/etherpad/state.tf (hunk 1 above)
Comment (cmharlow) [reply terraform/applications/etherpad/state.tf]: done

File: terraform/applications/etherpad/variables.tfvars (hunk 2 above)
Comment (cmharlow) [reply terraform/applications/etherpad/variables.tfvars]: sure removed tfvars then.

**Pull request merged**